lib/hergetto/helpers/service_start_helper.ex

```diff
@@ -1,10 +1,11 @@
 defmodule Hergetto.Helpers.ServiceStartHelper do
-  def start(supervisor, service) do
```

Comment:
Why was this one removed? Because we now have a lot of create statements with nil, which doesn't look nice.
We can keep both.

Reply:
We now have 1 statement with nil. I dont see the point in keeping the old function if you are just going to use it in one spot.